routers/api/v1/user/key.go

```diff
@@ -198,6 +199,11 @@ func GetPublicKey(ctx *context.APIContext) {
 
 // CreateUserPublicKey creates new public key to given user by ID.
 func CreateUserPublicKey(ctx *context.APIContext, form api.CreateKeyOption, uid int64) {
```

Comment:
[DELETE `/user/keys/{id}`](https://gitea.com/api/swagger#/user/userCurrentDeleteKey) is missing

Reply:
Done.